endpoints-support.md

```diff
@@ -20,6 +20,12 @@
 
 | Endpoints                                                                       | Verb     | V200 | V300 | V500 |V600 |V800
 | --------------------------------------------------------------------------------------- | -------- | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: |
+|     **Enclosures**                                                                                                                          |
```

Comment:
enclosure supports should be till api800

Reply:
changed.